dom.bs

```diff
@@ -6760,7 +6790,7 @@ method steps are:
 
  <li><p>If <var>attribute</var> is null, create an <a>attribute</a> whose
  <a for=Attr>local name</a> is <var>qualifiedName</var>, <a for=Attr>value</a> is
- <var>value</var>, and <a for=Node>node document</a> is <a>this</a>'s <a for=Node>node document</a>,
+ stringified <var>value</var>, and <a for=Node>node document</a> is <a>this</a>'s <a for=Node>node document</a>,
```

Comment:
As noted elsewhere, this doesn't work as-is.

Reply:
I actually rewrote all this from scratch, adding an algorithm that validates with TT and sets the attr value (a string at that point). There's no need for stringification in DOM now, PTAL.